src/locale/locales/ga/messages.po

```diff
@@ -511,109 +383,81 @@ msgstr "Cuir an taifead DNS seo a leanas le d'fhearann:"
 
 #: src/components/FeedCard.tsx:293
 msgid "Add this feed to your feeds"
-msgstr ""
+msgstr "Cuir an fotha seo le do chuid fothaí"
 
-#: src/view/com/profile/ProfileMenu.tsx:267
-#: src/view/com/profile/ProfileMenu.tsx:270
+#: src/view/com/profile/ProfileMenu.tsx:267 src/view/com/profile/ProfileMenu.tsx:270
 msgid "Add to Lists"
 msgstr "Cuir le liostaí"
 
 #: src/view/com/feeds/FeedSourceCard.tsx:266
 msgid "Add to my feeds"
 msgstr "Cuir le mo chuid fothaí"
 
-#: src/view/com/auth/onboarding/RecommendedFeedsItem.tsx:139
-#~ msgid "Added"
-#~ msgstr "Curtha leis"
-
-#: src/view/com/modals/ListAddRemoveUsers.tsx:192
-#: src/view/com/modals/UserAddRemoveLists.tsx:156
+#: src/view/com/modals/ListAddRemoveUsers.tsx:192 src/view/com/modals/UserAddRemoveLists.tsx:156
 msgid "Added to list"
 msgstr "Curtha leis an liosta"
 
 #: src/view/com/feeds/FeedSourceCard.tsx:125
 msgid "Added to my feeds"
 msgstr "Curtha le mo chuid fothaí"
 
-#: src/view/screens/PreferencesFollowingFeed.tsx:171
-#~ msgid "Adjust the number of likes a reply must have to be shown in your feed."
-#~ msgstr "Sonraigh an méid moltaí ar fhreagra atá de dhíth le bheith le feiceáil i d'fhotha."
-
-#: src/lib/moderation/useGlobalLabelStrings.ts:34
-#: src/lib/moderation/useModerationCauseDescription.ts:144
-#: src/view/com/modals/SelfLabel.tsx:76
+#: src/lib/moderation/useGlobalLabelStrings.ts:34 src/lib/moderation/useModerationCauseDescription.ts:144 src/view/com/modals/SelfLabel.tsx:76
```

Comment:
@surfdude29 You might know better since you look at these a lot. Shouldn't the different file paths be on separate lines here?

Reply:
@haileyok They are on separate lines when they're generated by Lingui, but Lingui seems to deal with them being on the same line OK too (which some software like POEdit does).

Also, if you look at the linter results as described [in this comment](https://github.com/bluesky-social/social-app/pull/5345#discussion_r1759844719), Lingui seems to compile the message catalogs perfectly fine as it shows 0 missing translations.

So I'm pretty sure it's A-OK 👌

Reply:
Amazing, thanks for the explanation!

Reply:
No worries :)